packages/wrangler/src/constellation/createProject.tsx

```diff
@@ -0,0 +1,64 @@
+import { Text, Box } from "ink";
```

Comment:
We're trying to move away from `ink`, in general. Can this be done without `ink`?

Reply:
https://github.com/cloudflare/workers-sdk/blob/9af1a640237ab26e6332e73e3656d16ca9a96e64/packages/wrangler/src/secret/index.ts has some examples

Reply:
Done.